policies/master_payer.json

```diff
@@ -0,0 +1,153 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "s3:Get*",
+                "s3:List*"
+            ],
+            "Resource": [
+                "arn:aws:s3:::${BillingBucketName}",
```

Comment:
@millsb, how do you want the `BillingBucketName` to appear in this policy so you can interpolate it on the frontend?

Reply:
The `${}` is problematic because that's valid javascript. I'd be okay with just using some existing template marker format. Like `{{ }}` or `<% %>` that we can pattern match against.

Reply:
Good call ... I will use `{{}}` in honor of Movember.